eZ/Publish/Core/MVC/Symfony/Matcher/ConfigurableMatcherFactoryInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * File containing the MatcherFactory interface.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\Matcher;
+
+interface ConfigurableMatcherFactoryInterface extends MatcherFactoryInterface
```

Comment:
This maybe should be marked internal? However if meant for custom code to be either used in or extended by, it should be a part of API or SPI respectively.

Reply:
Fixed